### DISCUSSION TITLE
Menanyakan nilai presisi

### DISCUSSION THREAD

**Discussion by nurilfm** [ledleledle/WebCBIR#7] [Q&A] [answered]
apakah semua gambar sama nilainya

**Comment by ledleledle** [comment, marked as answer]
What do you mean by "same"?
If you mean by calculation, you can rever to file `conf.csv`. Everytime you search image, the result will provide value **True Positive, False Positive, etc**. And for the calculation you can check to `runner.py` at line 41-44.
```
akurasi = round((((tn+tp)/(tn+tp+fn+fp))*100),2)
presisi = round(((tp/(fp+tp))*100),2)
rekal = round(((tp/(fn+tp))*100),2)
f1 = round((2*(rekal*presisi)/(rekal+presisi)),2)
```

Hopefully this answer will help

**Reply by ledleledle** [reply]
Oh bwt. Sorry I'm late lol
